### ISSUE TITLE
OP1058 Add support for order By District in group list display

### ISSUE THREAD

**Issue by nicktrosnant** [u3a-siteworks-development/u3a-siteworks-core#97]
Where the support for the District field is enabled in the u3a Settings (Venues tab) provide additional button on the front end to list groups sorted by District
![Screenshot from 2024-07-05 17-22-27](https://github.com/u3a-siteworks-development/u3a-siteworks-core/assets/34215865/cbf347cc-c9f9-4ccd-ae96-feacf5494e65)

**Comment by nicktrosnant** [comment]
Agreed that the fact that "District" is a free text field rather than a taxonomy will make this subject to the accuracy or otherwise of data entry by the WM.  We will be relying on them to be 100% consistent.

Also agree that as District is an attribute of a Venue it does mean a fairly convoluted set of queries to produce the listing.

If you are enjoying the challenge then by all means carry on, but if you think it doesn't justify the work involved then we can go back to the change board and suggest this is a "will not fix".   This would be a "Feature" so would not be deployed to users until the next Feature update - November-ish.

**Comment by nicktrosnant** [comment]
Dev team agreed to suggest alternative approach using group categories to change board.

**Comment by edwery** [comment]
The district is free-form text and not part of any taxonomy, so the data present could be anything. We would need to gather all possible districts by searching all
venues, then order them - which would be slow, and probably inaccurate as districts could be in any case, with or without spaces etc.

The district being an attribute of a venus also poses problems for the type of query we could use to build the list.

For these reasons we think the display is likely to be inaccurate and slow to display - so suggesting we do not implement.

**Comment by mike99christie** [comment]
Closing this issue
